test/doc/algo/detail/print.hpp

```diff
@@ -0,0 +1,29 @@
+//==================================================================================================
+/**
+   EVE - Expressive Vector Engine
+   Copyright : EVE Contributors & Maintainers
+   SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#pragma once
+
+#include <eve/module/core.hpp>
+#include <eve/algo/concepts/relaxed.hpp>
+#include <iostream>
+
+namespace eve::detail
```

Comment:
A bit wierd that this is in eve namespace but w/e

Reply:
where do you want to put it ?

Reply:
any namespace that is not eve? doc_utils namespace for example.

Reply:
changed doc_utils